etc/profile-m-z/QMediathekView.profile

```diff
@@ -27,29 +27,50 @@ include disable-programs.inc
 include disable-shell.inc
 include disable-xdg.inc
 
+whitelist ${HOME}/.config/QMediathekView
+whitelist ${HOME}/.local/share/QMediathekView
+
+whitelist ${HOME}/.config/mpv
+whitelist ${HOME}/.config/smplayer
+whitelist ${HOME}/.config/totem
+whitelist ${HOME}/.config/vlc
+whitelist ${HOME}/.config/xplayer
+whitelist ${HOME}/.local/share/totem
+whitelist ${HOME}/.local/share/xplayer
+whitelist ${HOME}/.mplayer
+
+whitelist ${DOWNLOADS}
+whitelist ${VIDEOS}
+
 whitelist /usr/share/qtchooser
```

Comment:
```suggestion
whitelist ${DOWNLOADS}
whitelist ${VIDEOS}

whitelist ${HOME}/.config/mpv
whitelist ${HOME}/.config/smplayer
whitelist ${HOME}/.config/totem
whitelist ${HOME}/.config/vlc
whitelist ${HOME}/.config/xplayer
whitelist ${HOME}/.local/share/totem
whitelist ${HOME}/.local/share/xplayer
whitelist ${HOME}/.mplayer
whitelist /usr/share/qtchooser
```

This makes it easier to sort everything and I think it's more similar to other
profiles.

Reply:
Looks better like that indeed. Thanks!